src/qibo/models/variational.py

```diff
@@ -27,51 +30,25 @@ class VQE:
             vqe = models.VQE(circuit, hamiltonian)
             # optimize using random initial variational parameters
             initial_parameters = np.random.uniform(0, 2, 1)
-            vqe.minimize(initial_parameters)
+            opt = CMAES()
+            best, params, _ = vqe.minimize(opt, initial_parameters, fit_options=options)
```

Comment:
I think `options` is not defined and still passed to the `fit_options` argument.